presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastoreConfig.java

```diff
@@ -0,0 +1,75 @@
+package com.facebook.presto.hive.metastore.thrift;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+
+import javax.validation.constraints.NotNull;
+
+public class ThriftHiveMetastoreConfig {
+
+  private final String PREFIX = "metastore";
+
+  private int maxRetries = 10;
+  private double backoffScaleFactor = 2.0;
+  private Duration minBackoffDelay = Duration.valueOf("1s");
+  private Duration maxBackoffDelay = Duration.valueOf("30s");
+  private Duration maxRetryTime = Duration.valueOf("30s");
+
+  @NotNull
+  public int getMaxRetries() {
```

Comment:
This formatting is incorrect per our code style and will fail the build. Please see https://github.com/airlift/codestyle

Reply:
OK